Countly.js

```diff
@@ -127,7 +127,7 @@ Countly.hasBeenCalledOnStart = function () {
  *
  * Used to send various types of event;
  *
- * @param {object} options event
+ * @param {CountlyEventOptions} options event
```

Comment:
this old statement is a little weird right? it sounds like a special 'options' event. lets make it event options and explain there are the possible values/keys you can provide to record an event

Reply:
done